File: api/models/paste_stats.go
package models

// Various statistics of a single paste.
type PasteStats struct {
	Stats
	Pasties map[string]Stats `json:"pasties"`
}

type Stats struct {
	Lines uint64 `json:"lines"`
	Words uint64 `json:"words"`
	Size  uint64 `json:"size"`
}

File: api/Models/PasteStats.cs
namespace pastemyst.Models;

public class PasteStats : Stats
{
    public Dictionary<string, Stats> Pasties { get; set; } = new();
}

public class Stats
{
    public int Lines { get; set; }
    public int Words { get; set; }
    public int Bytes { get; set; }
}